network/gossip/libp2p/test/testUtil.go

```diff
@@ -24,16 +24,47 @@ import (
 	"github.com/onflow/flow-go/utils/unittest"
 )
 
-var rootBlockID = unittest.IdentifierFixture().String()
+var rootBlockID string
+
+// allocatedPorts keeps track of ports allocated to different tests
+var allocatedPorts map[int]struct{}
```

Comment:
we will also need a mutex to protect this since tests from different packages may run concurrently

Reply:
thanks, addressed in https://github.com/onflow/flow-go/pull/99/commits/dbc419fc31bb0c6c89d7aac66c9697772399f4d0.